packages/bs4/src/services/tooltip.service.ts

```diff
@@ -124,6 +124,11 @@ const TRIGGER_MANUAL = "manual";
  * ------------------------------------------------------------------------
  */
 
+// Get the class of "this" in methods of TooltipService. (It could be TooltipService or the derived PopoverService.)
+const classOf = (that: TooltipService) => {
+  return that.constructor as typeof TooltipService;
+};
+
```

Comment:
Schöne Methode, bitte ins utils modul verschieben

Reply:
Hab ich hier gemacht:

https://github.com/ribajs/riba/commit/8ff5247a50388e94445c2ed2f285533e9cc15cb1
https://github.com/ribajs/riba/commit/f71a173ebff9cac08c1310a9b2a08a4a1e2dd1ab

Leider in der allgemeinen Form nur mit `any` hinbekommen (und ohne das `as typeof ...`).
Dachte, das müsste auch irgendwie mit einem Type-Parameter (hier dann `TooltipService` als Type-Parameter) für die Parent-Klasse zu machen sein. Aber kriege da dann TypeScript-Fehler wegen der "constructor"-Property für allgemeinen Typ.

Reply:
Das funktioniert auch nicht? https://stackoverflow.com/a/52183279/1465919

Reply:
Ich weiß nicht, wie das helfen kann.